Volume/Supporting/Main.swift

```diff
@@ -32,6 +34,25 @@ struct Main: App {
             announcementPath: Secrets.announcementsPath
         )
     }
+    
+    private func registerForPushNotifications() {
+        UNUserNotificationCenter.current().getNotificationSettings { settings in
+            if settings.authorizationStatus == .notDetermined {
+                UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .badge, .sound]) { granted, error in
+                    print("Push notification permissions granted: \(granted)")
```

Comment:
Not saying you should change this, but just wanted to verify you know this print statement is in here.  If you want to keep a print statement in production code that's up to you all, but pods I've been on previously try to limit print statements in production unless the feature will be worked on over a span of multiple PRs.

If you think you should keep it though go ahead.